rllib/tuned_examples/dqn/cartpole_dqn.py

```diff
@@ -18,11 +18,15 @@
         lr=0.0005 * (args.num_learners or 1) ** 0.5,
         train_batch_size_per_learner=32,
         replay_buffer_config={
-            "type": "PrioritizedEpisodeReplayBuffer",
+            "type": "EpisodeReplayBuffer",
```

Comment:
is this just for testing?

Reply:
Yeah, I wanted to check with you, if we proceed like this and then all buffers get the metrics. Then I can test with any of them.